02_bubble/paraganz/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+	cnt := len(s)
+	for i := 0; i < cnt; i++ {
+		for j := i + 1; j < cnt; j++ {
+			if s[i] > s[j] {
+				tmp := s[i]
+				s[i], s[j] = s[j], tmp
```

Comment:
You don't need `tmp`. Use `s[i]` directly.

Reply:
Done